core/src/main/java/bio/terra/pearl/core/service/participant/EnrolleeRelationService.java

```diff
@@ -127,4 +128,19 @@ public List<EnrolleeRelation> filterValid(List<EnrolleeRelation> enrolleeRelatio
         return enrolleeRelations.stream().filter(this::isRelationshipValid).collect(Collectors.toList());
     }
 
+    public List<EnrolleeRelation> findRelationsForFamily(UUID familyId) {
+        return filterValid(dao.findRelationsForFamily(familyId));
+    }
+
+    @Transactional
+    public void deleteByFamilyId(UUID id, DataAuditInfo info) {
+        List<EnrolleeRelation> relations = findRelationsForFamily(id);
+        bulkDelete(relations, info);
+    }
+
+    @Transactional
+    public void deleteByStudyEnvironmentId(UUID studyEnvironmentId, DataAuditInfo info) {
```

Comment:
this should not be an audited operation -- only occurs in the case of study deletion